edkrepo/commands/combo_command.py

```diff
@@ -52,11 +52,11 @@ def run_command(self, args, config):
                 sources = manifest.get_repo_sources(combo)
                 length = len(max([source.root for source in sources], key=len))
                 for source in sources:
-                    if source.branch:
-                        ui_functions.print_info_msg("    {} : {}".format(source.root.ljust(length), source.branch), header=False)
-                    elif source.commit:
+                    if source.commit:
                         ui_functions.print_info_msg("    {} : {}".format(source.root.ljust(length), source.commit), header=False)
-                    elif source.tag:
+                    elif source.tag and not source.commit:

```

Comment:
I think the second part of all of these elif conditions are redundant...
- `not source.commit` is implied already if we didn't pass into `if source.commit`
- Similarly, `not (source.commit or source.tag)` is implied if we didn't pass into `if source.commit` and `elif source.tag`

Reply:
Agreed. I felt this was a better way of documenting the priority order than adding a comment while also making the logic future proof if another conditional case were added.